templates/nginx-conf.j2

```diff
@@ -47,6 +47,12 @@ http {
     sendfile        on;
     #tcp_nopush     on;
 
+    {% if nginx_proxy_conf_http is defined %}
+      {% for http_conf_line in nginx_proxy_conf_http %}
+{{ http_conf_line }}
```

Comment:
Reason was an attempt to get rid of the indentation that appears in the resultant file, cf

![screen_shot_2017-07-31_at_11_34_40](https://user-images.githubusercontent.com/2727871/28774220-8598cbbe-75e4-11e7-83bc-7b357dca5122.png)

It worked at one point, then stopped working after my "if/else" commits. 

Any ideas how to fix that properly in the resultant file?